examples/Workflow/WorkflowConsoleApp/Program.cs

```diff
@@ -47,7 +47,16 @@
 using var host = builder.Build();
 host.Start();
 
-using var daprClient = new DaprClientBuilder().Build();
+DaprClient daprClient;
+string apiToken = Environment.GetEnvironmentVariable("DAPR_API_TOKEN");
+if (!string.IsNullOrEmpty(apiToken))
+{
+    daprClient = new DaprClientBuilder().UseDaprApiToken(apiToken).Build();
+}
+else
+{
+    daprClient = new DaprClientBuilder().Build();
+}
```

Comment:
We still need the `using`. After the initialization, you can just wrap the `while` in this:

```csharp
using (daprClient)
{
    // Loop here.
}
```